main.go

```diff
@@ -94,8 +103,16 @@ func main() {
 		dief("looking for test data: %v\n", err)
 	}
 	for _, name := range remove {
-		if err := os.RemoveAll(name); err != nil {
-			dief("removing %v: %v\n", name, err)
+		err := os.RemoveAll(name);
+		if err != nil {
+			failedToRemove[name] = err
+		}
+	}
+	
+	if len(failedToRemove) > 0 {
+		fmt.Fprintf(os.Stderr, "\n# Failed to remove below item, please remove manually\n")
+		for name, err := range failedToRemove {
```

Comment:
The purpose of this program is to provide a minimal version of Go. If deletion fails then the program has failed.

Here's how we solved a similar issue in another tool: https://github.com/golang/go/blob/master/misc/makerelease/makerelease.go#L801